src/libcore/num/mod.rs

```diff
@@ -3848,7 +3848,8 @@ fn from_str_radix<T: FromStrRadixHelper>(src: &str, radix: u32) -> Result<T, Par
 /// 
 /// # Potential causes
 ///
-/// Among other causes, `ParseIntError` can be thrown because of leading or trailing whitespaces in the string e.g. when it is getted from the standard input. 
+/// Among other causes, `ParseIntError` can be thrown because of leading or trailing whitespaces 
+/// in the string e.g. when it is getted from the standard input. 
```

Comment:
"getted' doesn't work here. Perhaps "when obtained from the standard input"?

Reply:
Yes, forgive my English. I'll write that down.